exercises/reverse-string/description.md

```diff
@@ -2,11 +2,15 @@
 
 The introductory exercise to reverse an input string.
 
-This is a typical first program for beginning programming in a new language
+This is a typical program for beginning programming in a new language
 or environment after being able to do simple tasks.
 
 The objectives are simple:
 
 - Write a function that reverses the input string.
+- Test various mixed case inputs and output the reverse of them.
 - Run the test suite and make sure that it succeeds.
 - Submit your solution and check it at the website.
+
+Comments: 
+If testing tools are available, a property that can be tested is to reverse string twice and see if the output is the original string.
```

Comment:
No, this is not what I meant. This file is solely for the student. When I said add this comment I meant in the JSON, where it is meant for an implementor of the exercise, as in all-your-base:

https://github.com/exercism/problem-specifications/blob/master/exercises/all-your-base/canonical-data.json#L4-L23

Also, testing tools are always available ;) But usually we only use example based testing. Therefore, when you move the comment into the JSON, make it read like this:

> If property based testing tools are available, a good property to test is `reverse(reverse(str)) == str`.

Reply:
updated it in last PR, although I'm not sure how to write code syntax in JSON. ``` may not work, it is highlighted red in file.

Reply:
Well, it needs to be a string of course…